editor/svgcanvas.js

```diff
@@ -218,6 +221,14 @@ var addSvgElementFromJson = this.addSvgElementFromJson = function(data) {
 	}
 	svgedit.utilities.assignAttributes(shape, data.attr, 100);
 	svgedit.utilities.cleanupElement(shape);
+
+	// Children
+	if(data.children) {
```

Comment:
Please follow code convention of putting a space after 'if' and before (

`if (data.children) {`